packages/conventional-changelog-core/README.md

```diff
@@ -64,6 +64,12 @@ Type: `number` Default: `1`
 
 How many releases of changelog you want to generate. It counts from the upcoming release. Useful when you forgot to generate any previous changelog. Set to `0` to regenerate all.
 
+##### skipUnstable
+
+Type: `boolean` Default: `false`
+
+If set, unstable release tags will be skipped. EG: x.x.x-rc.1
```

Comment:
> If set, unstable release tags will be skipped, e.g., x.x.x-rc.1

Reply:
and done. also rebased on current master.